src/com/company/Main.java

```diff
@@ -21,49 +23,51 @@ public static void main(String[] args) {
         int water = waterSum / waterCup;
         int beans = beansSum / beansCup;
 
-        int min;
-        if (beans < milk && beans < water)
-        {
-            min = beans;
-        }
-        else if (milk < beans && milk < water)
-        {
-            min = milk;
-        }
-        else
-        {
-            min = water;
-        }
+        int min = minIng(milk, water, beans);
 
         System.out.println("Write how many cups of coffee you will need:");
         int cup;
 
-        while(min!=0)
-        {
+        while (min != 0) {
 
             cup = s.nextInt();
 
-            if (min >0 && min>cup)
-            {
+            if (min > 0 && min > cup) {
                 System.out.println("Yes, I can make that amount of coffee (and even " + (min - cup) + " more than that)");
                 min -= cup;
-            }
-            else if(min==1 && min==cup)
-            {
+            } else if (min == cup) {
                 System.out.println("Yes, I can make that amount of coffee");
                 min -= cup;
-            }
-            else
-            {
+            } else {
                 System.out.print("No, I can make only " + min + " cup(s) of coffee\n");
             }
-            waterSum -= cup*waterCup;
-            milkSum -= cup*milkCup;
-            beansSum -= cup*beansCup;
+            waterSum -= cup * waterCup;
+            milkSum -= cup * milkCup;
+            beansSum -= cup * beansCup;
 
         }
 
 
+    }
 
+    public static int minIng(int n1, int n2, int n3) {
```

Comment:
@EduardKhachatryan95 please add comments, and write more meaning fields or function names